tests/components/google_assistant/test_smart_home.py

```diff
@@ -488,76 +488,41 @@ async def test_execute(
     events = async_capture_events(hass, EVENT_COMMAND_RECEIVED)
     service_events = async_capture_events(hass, EVENT_CALL_SERVICE)
 
-    with patch.object(
```

Comment:
Since the tests already capture `EVENT_CALL_SERVICE`, I took out the `async_call` patching as we already have coverage that service calls fire events so it shouldn't be needed to check that as well here.